### PR TITLE
[Feature]: Settings modal dialog HTML markup

### PR DESCRIPTION
**Pull request by tux255**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import settingsIcon from './assets/settings.svg'
+import React, { useState }  from 'react';
```
Comment (Nedilko) [inline src/App.jsx:1]: `React` is redundant

File: src/components/Modal.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```
Comment (Nedilko) [inline src/components/Modal.jsx:1]: `React` is redundant

File: src/settings.js [hunk 3]
```diff
@@ -12,4 +12,37 @@ const TARGET_DATE = {
...
+    13: '06:00',
+    14: '06:30',
+    15: '07:00',
+    16: '07:30',
+    17: '08:00',
+    18: '08:30',
+    19: '09:30',
+    20: '10:00'
+}
+
```
Comment (Nedilko) [inline src/settings.js:47]: redundant?

Comment (tux255) [reply src/settings.js]: yes redunant. Will use Dayjs weekdays and hours list.

**Review by Nedilko — APPROVED** [review]

**Pull request merged**